playwright/test/helpers/contexts.ts

```diff
@@ -84,9 +84,7 @@ const stageApiToggleCookie = createCookie({
   value: 'true',
 });
 
-// TODO: context.addCookies should run for the first test of a suite (even on beforeAll/beforeEach)
```

Comment:
This TODO is 3 yrs old, and we don't want all scenarios to have cookies added, so removing.